readme.md

```diff
@@ -1,2 +1,75 @@
 React Material Icons
 ====================
+
+This package includes all [Material Icons](https://material.io/resources/icons/?style=baseline) as reusable React components
+
+## Table of content
+- [Usage](#usage)
```

Comment:
Please add consistent empty lines before and after titles and paragraphs like in the other packages: https://raw.githubusercontent.com/aboutbits/pagination/master/readme.md